readme.md

```diff
@@ -548,12 +548,46 @@ to remove it from the callback chain if you don't want it to, e.g.
 **Note that if you previously added a custom dimension extraction callback to the *amp_extract_image_dimensions* filter,
 you need to update it to hook into the *amp_extract_image_dimensions_batch* filter instead and iterate over the key value
 pairs in the single argument as per the example above.**
+
 ## Analytics
 
+There are two options you can follow to include analytics tags in your posts.
 
-To output proper analytics tags, you can use the `amp_post_template_analytics` filter:
+### Plugin Analytics Options
 
-```
+The plugin defines an analytics option to enable the addition of
```

Comment:
The links in the image references should be changed to the master branch before merging.